--- scan.nginx.conf.template.orig	2018-09-19 11:49:58.000000000 -0400
+++ scan.nginx.conf.template	2018-09-19 12:00:55.000000000 -0400
@@ -2,6 +2,7 @@
 # HEAD, POST /api/bom-import
 # HEAD, POST /v1/api/bom-import
 
+set $scan_upstream ${HUB_SCAN_HOST};
 location ~ ^/(api/bom-import|api/v1/bom-import)/?$ {
     client_max_body_size 3072m;
     client_body_timeout 3600s;
@@ -10,7 +11,7 @@
     }
     proxy_connect_timeout 60;
     proxy_intercept_errors on;
-    proxy_pass https://${HUB_SCAN_HOST}:${HUB_SCAN_PORT};
+    proxy_pass https://$scan_upstream:${HUB_SCAN_PORT};
     proxy_pass_header X-Host;
     proxy_read_timeout 3600;
     proxy_redirect off;
@@ -46,7 +47,7 @@
     }
     proxy_connect_timeout 60;
     proxy_intercept_errors on;
-    proxy_pass https://${HUB_SCAN_HOST}:${HUB_SCAN_PORT};
+    proxy_pass https://$scan_upstream:${HUB_SCAN_PORT};
     proxy_pass_header X-Host;
     proxy_read_timeout 3600;
     proxy_redirect off;
@@ -77,7 +78,7 @@
     }
     proxy_connect_timeout 60;
     proxy_intercept_errors on;
-    proxy_pass https://${HUB_SCAN_HOST}:${HUB_SCAN_PORT};
+    proxy_pass https://$scan_upstream:${HUB_SCAN_PORT};
     proxy_pass_header X-Host;
     proxy_read_timeout 3600;
     proxy_redirect off;
@@ -109,7 +110,7 @@
     }
     proxy_connect_timeout 60;
     proxy_intercept_errors on;
-    proxy_pass https://${HUB_SCAN_HOST}:${HUB_SCAN_PORT};
+    proxy_pass https://$scan_upstream:${HUB_SCAN_PORT};
     proxy_pass_header X-Host;
     proxy_read_timeout 3600;
     proxy_redirect off;
@@ -141,7 +142,7 @@
     }
     proxy_connect_timeout 60;
     proxy_intercept_errors on;
-    proxy_pass https://${HUB_SCAN_HOST}:${HUB_SCAN_PORT};
+    proxy_pass https://$scan_upstream:${HUB_SCAN_PORT};
     proxy_pass_header X-Host; 
     proxy_read_timeout 3600;
     proxy_redirect off;
