--- upload.nginx.conf.template.orig	2018-09-21 09:50:04.000000000 -0400
+++ upload.nginx.conf.template	2018-09-21 09:50:04.000000000 -0400
@@ -2,6 +2,7 @@
 # * /upload/*
 # POST api/upload
 
+set $upload_upstream ${HUB_UPLOAD_CACHE_HOST};
 location ~ ^/(api/uploads)/?$ {
     client_max_body_size 3072m;
     client_body_timeout 3600s;
@@ -17,7 +18,7 @@
 
     proxy_connect_timeout 60;
     proxy_intercept_errors on;
-    proxy_pass https://${HUB_UPLOAD_CACHE_HOST}:${HUB_UPLOAD_CACHE_PORT};
+    proxy_pass https://$upload_upstream:${HUB_UPLOAD_CACHE_PORT};
     proxy_pass_header X-Host;
     proxy_read_timeout 3600;
     proxy_redirect off;
